install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

```diff
@@ -76,6 +76,19 @@ spec:
 {{- end }}
           - name: GF_PATHS_DATA
             value: /data/grafana
+{{- if or $.Values.env $.Values.envSecrets }}
```

Comment:
This line seems redundant? i think the range will just never produce anything if both are false anyways?

Reply:
Hi Howard, thanks, change has been made and PR updated.